tools/sof-kernel-log-check.sh

```diff
@@ -261,6 +261,9 @@ ignore_str="$ignore_str"'|asix .-.+\..:.\.. en.+: Failed to .+'
 # https://github.com/intel-innersource/drivers.audio.ci.sof-framework/issues/246
 ignore_str="$ignore_str"'|mei_me 0000:00:16\..\: .+'
 
+# ignore the ACPI error on LNL.
+ignore_str="$ignore_str""|kernel: ACPI: \\\\: Can't tag data node"
```

Comment:
Do we have any idea what this error means? We do use ACPI. If yes please add a comment. Is it safe to ignore across all products? If not there is a `lnl` platform section below.

Reply:
The errors come from https://github.com/thesofproject/linux/blob/bf67942088295fc31042c8d8d71a212c53a9134c/drivers/acpi/property.c#L366C1-L386C2.  
It means that data nodes failed to tie to ACPI handles. I don't know much about this, but from my perspective it's harmless to audio.